### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by reubenjnazareth**
Added a new branch for Avatar Component and completed avatar component.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -71,25 +72,86 @@ logo {
...
     bottom: 0; 
 }
 
+.wrapper {
+    max-width: ;
+    background: var(--secondary-color);
+    box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.05);
+    order-radius: 5px;
+    padding: 1rem;
+    margin: 1rem auto;
```
Comment (sharath-io) [inline styles.css:96]: Good to see you have used rem.
 I suggest you to use rem than px. check this throughout code.

File: index.html [hunk 2]
```diff
@@ -23,7 +23,7 @@ <h1 class="logo">BEN-UI</h1>
     <div class="introduction">
         <h3>A set of small CSS componenets that will assist you in making your websites quickly.</h3> 
         <img class="introduction-img" src="images/introduction img.svg" width="400" 
-        height=200" />
+        height="200" />
```
Comment (sharath-io) [inline index.html]: Its not suggestible to use inline styling bring them to css file.

**Review by sharath-io — CHANGES_REQUESTED** [review]
Suggested few changes. Can update those and you are good to go

File: styles.css [hunk 3]
```diff
@@ -20,12 +20,15 @@ body {
     padding: 0 8px;
     background: var(--primary-color);
     font-size: larger;
-    color: white;
+    color: var(--secondary-color);
+    box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.05);
```
Comment (adith-a-danthi) [inline styles.css]: If you're using the same shadow in multiple places you can create a CSS variable for it

File: styles.css [hunk 4]
```diff
@@ -71,25 +72,86 @@ logo {
...
+}
+
+iframe {
+    
+    margin-left: 30px;
+}
+
+hr {
+    border-color: var(--secondary-color);
+    margin: 10px 30px 10px 30px ;
```
Comment (adith-a-danthi) [inline styles.css]: You can simplify this to `margin: 10px 30px`. Also it's better to use `rem` instead of `px`.

File: avatar.html [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+<div class="component-title">AVATAR</div>
+<div class="component-intro">Avatars are images that users can set as their profile picture.</div>
+
+
+<div class="sub-title">Circle Avatars</div>
+<div class="avatar-nav">
+    <img src="images/avatarImage1.jpeg" class="avatar-circle-large avatar">
+    <img src="images/avatarImage1.jpeg" class="avatar-circle-medium avatar">
+    <img src="images/avatarImage1.jpeg" class="avatar-circle-small avatar">
```
Comment (adith-a-danthi) [inline avatar.html]: Add `alt` attribute to `` tags

**Review by adith-a-danthi — COMMENTED** [review]
Few suggested changes
- Add `alt` attribute to `img` elements
- Use relative units like `rem/em` instead of `px`.
- Move inline styles to CSS classes

File: styles.css (hunk 1 above)
Comment (reubenjnazareth) [reply styles.css:96]: made the changes.

File: styles.css (hunk 4 above)
Comment (reubenjnazareth) [reply styles.css]: thanks. changes updated.

**Review by sharath-io — APPROVED** [review]

File: avatar.html (hunk 5 above)
Comment (reubenjnazareth) [reply avatar.html]: Updated!

**Pull request merged**